contracts/utils/BridgeMessage.sol

```diff
@@ -163,12 +142,7 @@ library BridgeMessage {
 
         // extract target address from payload (35-54)
         address targetAddress;
-        // why `add(targetAddressLength, offset)`?
-        // At this point, offset = 35, targetAddressLength = 20. `mload(add(payload, 55))`
-        // reads the next 32 bytes from bytes 23 in paylod, because the first 32 bytes
-        // of payload stores its length. So in reality, bytes 23 - 54 is loaded. During
-        // casting to address (20 bytes), the least sigificiant bytes are retained, namely
-        // `targetAddress` is bytes 35-54
```

Comment:
can we keep these comments? also line 120-127 and line 185-190. I hope it's useful for future readers

Reply:
sure! I'll add them back in.